src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FILE_NOT_WRITTEN_ERROR_MESSAGE = "Can't write data to the file: ";
+
+    @Override
+    public void write(String data, String toFileName, String path) {
+        try (BufferedWriter bufferedReader = new BufferedWriter(
+                new FileWriter(path + toFileName))) {
```

Comment:
The FileWriter is not being used in a try-with-resources statement. This can lead to resource leaks if writing to the file fails. Consider using try-with-resources to ensure the FileWriter is closed properly.

Reply:
++